### PR TITLE
Change export page layout

### PR DESCRIPTION
**Pull request by AdityaKhatri**

### CODE REVIEW

File: src/utils/request/index.ts [hunk 1]
```diff
@@ -263,6 +263,7 @@ function useRequest<T>(
 
     // NOTE: timestamp is used to re-trigger fetch
     const [timestamp, setTimestamp] = useState(() => {
+        console.warn('here', autoTrigger);
```
Comment (samshara) [inline src/utils/request/index.ts:266]: remove this

File: src/views/Export/ExportSelection/FilterForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+    ] = useRequest<LeadOptions>({
+        url: 'server://lead-options/',
+        query: {
+            projects: [projectId],
+        },
+        autoTrigger: true,
+        method: 'GET',
+        schemaName: 'projectLeadFilterOptions',
+        onSuccess: (response) => {
+            console.warn('here', response);
```
Comment (samshara) [inline src/views/Export/ExportSelection/FilterForm/index.tsx]: i think this is unnecessary.

File: src/views/Export/LeadsTable/index.tsx [hunk 3]
```diff
@@ -119,15 +186,92 @@ function ExportLeadsTable(props: ComponentProps) {
...
+
+            const isAsc = tmpActiveSort?.charAt(0) !== '-';
+
+            const isCurrentHeaderSorted = tmpActiveSort === key
+                || (tmpActiveSort?.substr(1) === key && !isAsc);
+
+            if (isCurrentHeaderSorted) {
+                tmpActiveSort = isAsc ? `-${key}` : key;
+            } else {
+                tmpActiveSort = headerData?.defaultSortOrder === 'dsc' ? `-${key}` : key;
```
Comment (samshara) [inline src/views/Export/LeadsTable/index.tsx]: remove optional chaining here

File: src/views/Export/ExportSelection/FilterForm/index.tsx [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+                ...faramValues,
+                confidentiality: undefined,
+            };
+
+            isFilterEmptyValue = doesObjectHaveNoData(newFilter, ['']);
+
+            newFilter = {
+                ...faramValues,
+                confidentiality: ['unprotected'],
+            };
```
Comment (samshara) [inline src/views/Export/ExportSelection/FilterForm/index.tsx]: what is  use of this?

Comment (AdityaKhatri) [reply src/views/Export/ExportSelection/FilterForm/index.tsx]: For the viewer role, only unprotected data is available, that is why if filterOnlyUnproteted is true, then we set it my default.

Comment (samshara) [reply src/views/Export/ExportSelection/FilterForm/index.tsx]: here the created newFilter is unused.

**Pull request merged**